tests/basictests/jupyterhub.sh

```diff
@@ -9,14 +9,25 @@ source ${MY_DIR}/../util
 JH_LOGIN_USER=${OPENSHIFT_USER:-"admin"} #Username used to login to JH
 JH_LOGIN_PASS=${OPENSHIFT_PASS:-"admin"} #Password used to login to JH
 OPENSHIFT_LOGIN_PROVIDER=${OPENSHIFT_LOGIN_PROVIDER:-"htpasswd-provider"} #OpenShift OAuth provider used for login
-JH_AS_ADMIN=${JH_AS_ADMIN:-"true"} #Expect the user to be Admin in JupyterHub
 ODS_CI_REPO_ROOT=${ODS_CI_REPO_ROOT:-"${HOME}/src/ods-ci"}
 
 os::test::junit::declare_suite_start "$MY_SCRIPT"
 
+function setup_jupyterhub_groups() {
+    os::cmd::expect_success "oc project ${ODHPROJECT}"
+    header "Setting up JupyterHub Groups authentication"
+    jh_pod_name=$(oc get pods -l deploymentconfig=jupyterhub -o json | jq -r '.items[0].metadata.name')
+    JUPYTERHUB_GROUPS_DEFAULT_DENIED=$(oc exec ${jh_pod_name} -c jupyterhub -- env | grep JUPYTERHUB_GROUPS_DEFAULT_DENIED | cut -f2 -d'=')
```

Comment:
Seems like the test run had a problem with this line....not sure what's being returned.